server/models/witnesses.js

```diff
@@ -5,6 +5,6 @@ const schema = dataTypes => ({
 });
 
 module.exports = (sequelize, DataTypes) => {
-  const incidentWitnesses = sequelize.define("incidentWitnesses", schema(DataTypes));
-  return incidentWitnesses;
+  const Witnesses = sequelize.define("Witnesses", schema(DataTypes));
```

Comment:
combine and just do:

`return sequelize.define("Witnesses", schema(DataTypes));`